### DISCUSSION TITLE
Is there any plan to design the debug widget as same as vscode?

### DISCUSSION THREAD

**Discussion by meng-jpg** [eclipse-theia/theia#10384] [Q&A]
It means to display the stake frames of all the threads at the same time.

<img width="438" alt="截屏2021-11-05 上午8 52 35" src="https://user-images.githubusercontent.com/74631407/140440260-85dad69f-92a8-4c12-bf80-a64499d5b155.png">

**Comment by vince-fugnitto** [comment]
@meng-jpg thank you for the discussion :+1: 
I believe when it was implemented it was by design to have a dedicated `threads` and `call stack` sections as part of the debug-view (in vscode it seems that they combine the two). Some previous discussions were had at https://github.com/eclipse-theia/theia/issues/8305.
